### ISSUE TITLE
Naga is too easy

### ISSUE THREAD

**Issue by Sandman25DCSS** [jeremygurr/dcssca#92]
I  believe Naga is too easy currently, it does not need so huge buffs to compensate for slow speed, especially since normal speed characters cannot avoid fights often too, thanks to changed stairdancing and movement penalty for kiting/pillardancing.
I am at XL 9 and I have AC+7, that's more than Gargoyle (+5) while having much more HP (+20% vs -20%) and no deformed body either. So overall I have AC 19 in ring mail while I would have only AC 12 in standard crawl. Also spit poison 3 is OP, Centaur dies to a single evocation in 2 turns, even Wyvern got to "almost dead" while approaching me.

**Comment by jeremygurr** [comment]
I agree. I haven't done anything to nerf them yet to counter balance the buffs, so it is understandable they aren't properly balanced yet. 

And the game has evolved significantly since I made that change to the Naga, which was really meant to make them more approachable for beginners. But now that we have easy mode, it's not as big of a deal.

What if I revert all naga changes back to original, except I allowed naga to get spit poison 2 at level 6, and spit poison 3 at level 12? Would that still be too powerful to be fun?
